src/services/twilio-api/get-help-doc-link.js

```diff
@@ -0,0 +1,60 @@
+const jsonMap = require('../api-doc-mapping.json');
+// url: {path : {topics: {}, commands: {}}, subpath : {wherever applicable}}
```

Comment:
Needs to be removed? If it was deliberately put up, could you please add a pointer what it describes?

Reply:
It's supposed to explain the json structure. I can mention "json structure" to the comment ?